### PR TITLE
Azureadapplications

### PR DESCRIPTION
**Pull request by paulculmsee**
---
When submitting a connector, please check the following conditions for your PR.

- [x] I attest that the connector works and I verified by deploying and testing all the operations.
- [x ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x ] I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`.

### CODE REVIEW

File: independent-publisher-connectors/AzureADApplications/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          "x-ms-apimTemplateParameter.value": "eventual",
+          "x-ms-apimTemplateParameter.existsAction": "override",
+          "x-ms-apimTemplate-policySection": "Request",
+          "x-ms-apimTemplate-operationName": [
+            "ListApplications"
+          ]
+        }
+      }
+    ],
+    "publisher": "Paul Culmsee"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/AzureADApplications/apiProperties.json:59]: @paulculmsee Please add `stackOwner` as well.

Documentation - [Adding metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-4-add-metadata)

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/AzureADApplications/apiProperties.json (hunk 1 above)
Comment (paulculmsee) [reply independent-publisher-connectors/AzureADApplications/apiProperties.json]: Sorry - corrected and pushed

**Review by sriyen-msft — APPROVED** [review]

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
Deployment Pending..

**Comment by ccpauto** [comment]
[[certify-connector]]

**Comment by mamurshe** [comment]
[[certify-connector]]

File: independent-publisher-connectors/AzureADApplications/apiProperties.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/AzureADApplications/apiProperties.json:8]: @paulculmsee We have a practice to keep the client id value as `"[[DUMMY]]"`. Can you please update it accordingly?

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
@paulculmsee I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have, Thanks.

**Comment by paulculmsee** [comment]
Sorry was not working on this one while there were discussions on whether it will be IP vs MS supported. Will get back this in the next couple of days

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
@paulculmsee Can you please provide step-by-step instructions (with screenshots, if possible) to set up an OAuth app for your connector? Everything else looks good to move forward.

**Comment by sriyen-msft** [comment]
@paulculmsee We are prepping your connector production deployment; we need to set up an OAuth app and add its client id and secret values to our Production KeyVault. We are unable to set up an OAuth app using our test account as it doesn’t have the required permissions.  

Since you are an internal partner, can you please set up the OAuth app and share the client id and secret to us in a secure channel, so that we can upload them to our Prod KV to proceed with production deployment.

@natalie-pienkowska and @mamurshe  - FYI

**Comment by natalie-pienkowska** [comment]
Our team is currently working on setting up OAuth for this connector. We will provide updates and comments along the way.

**Comment by natalie-pienkowska** [comment]
Hi @paulculmsee we are currently working on getting your connector on the next deployment train. We will let you know if there are any issue.

**Comment by natalie-pienkowska** [comment]
Unfortunately, we had some issues with the deployment. @mamurshe is on top of this and will do everything he can to deploy these connectors.

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
@paulculmsee Apologies for the delay in response, We are actively working on preparing and deploying your connector to publish environments. As you know that we are deploying the connector as a Standard connector, For that, we need to go thru a few guidelines and approvals from the concerned teams (ex. Power BI API team) which is a bit lengthy process. Please allow us some more time, we will keep you posted with the updates and deployment plan soon. Thank you so much for your patience.

**Comment by Amjed-Ayoub** [comment]
Dear partner,

We apologize for the delay you might be experiencing while we are working on a high volume of connector requests. Rest assured that we are doing everything we can to address this volume. We are taking measures like improving our processes and hiring additional team members to guarantee resolving your connector requests in a timely manner. 

Please let us know if you have any questions in the meantime.

File: independent-publisher-connectors/AzureADApplications/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,500 @@
...
+  },
+  "parameters": {
+  },
+  "securityDefinitions": {
+    "undefined": {
+      "type": "oauth2",
+      "flow": "accessCode",
+      "authorizationUrl": "https://login.windows.net/common/oauth2/authorize",
+      "tokenUrl": "https://login.windows.net/common/oauth2/authorize",
+      "scopes": {}
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/AzureADApplications/apiDefinition.swagger.json:478]: Hi @paulculmsee I hope you are doing well.
Apologies for the delay in the process.
Can you please add the scopes that you are targeting the Graph API for, and be more specific?
Thank you very much.

Comment (Amjed-Ayoub) [inline independent-publisher-connectors/AzureADApplications/apiDefinition.swagger.json:478]: Hi @paulculmsee I hope you are doing well.
Apologies for the delay in the process.
Can you please add the scopes that you are targeting the Graph API for, and be more specific?
Thank you very much.

File: independent-publisher-connectors/AzureADApplications/apiProperties.json [hunk 4]
```diff
@@ -0,0 +1,62 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "[[DUMMY]]",
+          "scopes": [],
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/AzureADApplications/apiProperties.json:9]: Please add the specific scopes. Thank you very much.

Comment (Amjed-Ayoub) [inline independent-publisher-connectors/AzureADApplications/apiProperties.json:9]: Please add the specific scopes. Thank you very much.

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

**Comment by Amjed-Ayoub** [comment]
Hello @paulculmsee,
I hope you are doing well.
Apologies for the delay in progress. 
Your connector is in production deployment and queued up for the next deployment schedule. This process typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps.
Please let us know if you have any questions.
Thank you very much for working with us.

**Review by sriyen-msft — APPROVED** [review]

**Comment by sriyen-msft** [comment]
@paulculmsee Apologies for the delay in response. 

Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Pull request merged**